README.md

```
@@ -58,18 +58,19 @@ order.AddRow(
-preparedPayment, err := c.PreparePayment(order)
+response, err := c.PreparePayment(order)
 if err != nil {
   //Handle error..
 }
 
 //Get the url to where the payment can be completed
-url := preparedPayment.URL()
+url := response.PreparedPayment.URL()
 ```
 
 ## Prepared payment callback (decode payment response)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)